### PR TITLE
Final backports for Julia 1.3.1 (v2)

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```
Comment (fredrikekre) [inline Project.toml]: !

Comment (DilumAluthge) [reply Project.toml]: Added in https://github.com/JuliaLang/Pkg.jl/pull/1543. Maybe an accident?

**Comment by KristofferC** [comment]
`final-final-ok-this-is-the-final.zip`

**Comment by fredrikekre** [comment]
Classic.

**Comment by fredrikekre** [comment]
Download failure on one worker, for some reason I can't restart...

**Pull request merged**